client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -263,6 +268,17 @@ class Upload extends React.Component {
 		);
 	}
 
+	renderNotAvailable() {
+		return (
+			<EmptyContent
+				title={ this.props.translate( 'Not available for multi site' ) }
+				line={ this.props.translate( 'Use the WP Admin interface instead' ) }
+				action={ this.props.translate( 'Open WP Admin' ) }
+				actionURL={ this.props.selectedSite.options.admin_url }
```

Comment:
Can we do this via a selector in `connect`? There's [`getSiteAdminUrl()`](https://github.com/Automattic/wp-calypso/blob/4f2ec264ae3038c5cbf0f5c537bca463ceb72750/client/state/sites/selectors.js#L938-L956). (I hope we can get rid of passing the `selectedSite` object prop around in the long run.)